packages/frontend/src/pages/MetricFlow.tsx

```diff
@@ -302,6 +315,19 @@ const MetricFlowPage = () => {
                         error={metricFlowQueryResultsQuery.error}
                     />
                 </CollapsableCard>
+                <MetricFlowSqlCard
+                    projectUuid={activeProjectUuid}
+                    status={metricFlowQueryResultsQuery.status}
+                    sql={metricFlowQueryResultsQuery.data?.query.sql}
+                    error={metricFlowQueryResultsQuery.error}
+                    canRedirectToSqlRunner={user.data?.ability?.can(
+                        'manage',
+                        subject('SqlRunner', {
+                            organizationUuid: user.data?.organizationUuid,
```

Comment:
FYI this `organizationUuid: user.data?.organizationUuid` check  is redundant. This is also happening in other places in the FE and we have a ticket to eventually refactor this. (It should not be a hug risk while users have only 1 org) . You can leave it as it is.